### PR TITLE
Solution to error handling excercise

### PR DESCRIPTION
**Pull request by tkerdonc**

### CODE REVIEW

**Comment by tkerdonc** [comment]
from https://exercism.io/my/solutions/c7f5e714c866478389394ff7fda91ec4

**Comment by tkerdonc** [comment]
Exercise subject 
https://github.com/tkerdonc/go-training/tree/master/go/error-handling

File: go/error-handling/error_handling.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//This package provides functions
+package erratum
+
+// Use takes a ResourceOpener function as a parameter and attempts to
+// call it to obtain a resource. It resists TransientError generated by
+// this call.
+// It also uses this resource's Frob function, and repairs FrobErrors
+// generated by it by calling its Defrob function.
+func Use(o ResourceOpener, inp string) (returnedError error) {
+	returnedError = nil
```
Comment (ivajloip) [inline go/error-handling/error_handling.go]: I don't think that this is needed as the default value for error is `nil`.

File: go/error-handling/error_handling.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+//This package provides functions
+package erratum
+
+// Use takes a ResourceOpener function as a parameter and attempts to
+// call it to obtain a resource. It resists TransientError generated by
+// this call.
+// It also uses this resource's Frob function, and repairs FrobErrors
+// generated by it by calling its Defrob function.
+func Use(o ResourceOpener, inp string) (returnedError error) {
```
Comment (ivajloip) [inline go/error-handling/error_handling.go]: When I see `o` I feel like it's a bit too short. This is more of a feeling than an objective evaluation, however. The general rule is - the shorter the lifetime, the shorter the name could be (here the lifetime is short, so maybe one letter is acceptable).
As for `inp`, I would prefer either `in` or `input`, but then again, not a thing I feel very strongly about.

**Review by ivajloip — APPROVED** [review]
Two minor comments (I always try to find even the slightest remarks, but don't feel like your code is not good enough, I just try to use the chance to share my preferences), otherwise it's good :slightly_smiling_face:

File: go/error-handling/error_handling.go (hunk 1 above)
Comment (tkerdonc) [reply go/error-handling/error_handling.go]: Arf, yeah, eventually, I'll learn to start trusting the language on initialization values ^^. Thanks !

File: go/error-handling/error_handling.go (hunk 2 above)
Comment (tkerdonc) [reply go/error-handling/error_handling.go]: Never really thought about lifetime vs variable name size, but that sounds reasonable.

**Comment by tkerdonc** [comment]
This last push integrates your remarks. Thanks a lot !

**Pull request merged**